src/KeyringController.test.ts

```diff
@@ -360,62 +377,76 @@ describe('KeyringController', function () {
       expect(allAccounts).toStrictEqual(expectedAllAccounts);
     });
 
-    it('should add HD Key Tree without mnemonic passed as an argument', async function () {
+    it('should add HD Key Tree without mnemonic passed as an argument', async () => {
       const previousAllAccounts = await keyringController.getAccounts();
       expect(previousAllAccounts).toHaveLength(1);
-      const keyring = await keyringController.addNewKeyring('HD Key Tree');
-      const keyringAccounts = await keyring.getAccounts();
+      const keyring = await keyringController.addNewKeyring(KeyringType.HD);
+      const keyringAccounts = await keyring?.getAccounts();
       expect(keyringAccounts).toHaveLength(1);
       const allAccounts = await keyringController.getAccounts();
       expect(allAccounts).toHaveLength(2);
     });
 
-    it('should add HD Key Tree with mnemonic passed as an argument', async function () {
+    it('should add HD Key Tree with mnemonic passed as an argument', async () => {
       const previousAllAccounts = await keyringController.getAccounts();
       expect(previousAllAccounts).toHaveLength(1);
-      const keyring = await keyringController.addNewKeyring('HD Key Tree', {
+      const keyring = await keyringController.addNewKeyring(KeyringType.HD, {
         numberOfAccounts: 2,
         mnemonic: walletTwoSeedWords,
       });
-      const keyringAccounts = await keyring.getAccounts();
+      const keyringAccounts = await keyring?.getAccounts();
```

Comment:
Hmm. If we can't guarantee `addNewKeyring` returns a keyring object then what is the point of this test?